File: book-application/src/main/java/com/example/bookapplication/models/Book.java
package com.example.bookapplication.models;

import javax.persistence.*;

@Entity
@Table(name = "book_names")
public class Book {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column
    private long bookid;

    @Column
    private String title;

    @Column
    private String author;

    @Column
    private int release;

    public Book() { }

    public void setBookid(long bookid) {
        this.bookid = bookid;
    }

    public long getBookid() {
        return bookid;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getTitle() {
        return title;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getAuthor() {
        return author;
    }

    public void setRelease(int release) {
        this.release = release;
    }

    public int getRelease() {
        return release;
    }

}

File: react-app-client/src/components/Book.js
import React from 'react';

function Book(props) {
  return (
    <div>
      <h1>Title of book: { props.title }</h1>
      <h2>Author of book: { props.author }</h2>
      <h3>Relase date of book: { props.release }</h3>
    </div>
  )
}

export default Book;
